src/instance/mgn.go

```diff
@@ -53,13 +53,18 @@ func StartInstance(name string, configuration int64) error {
 	s := settings.GetSettings()
 
 	// write config
-	if err := writeConfig(config); err != nil {
+	iniServerCfg, iniEntryList, err := writeConfig(config)
+
+	if err != nil {
 		return util.OpError{3, "Error writing configuration"}
 	}
 
+	// force server_cfg and entry_list ini paths
 	// start
-	cmd := exec.Command(filepath.Join(s.Folder, s.Executable), strings.Split(s.Args, " ")...)
-	now := strings.Replace(time.Now().String(), " ", "_", -1)
+	// FIXME: s.Args has been  discarted. No real use so far?
```

Comment:
Agree. Go is totally new for me (those are my first attempts on it). I've been fooled by exec.Command enough to discard joining ini parameters with s.Args. :o)
Will try to add it after refactor zip thing.